src/NuGet.Core/NuGet.Build.Tasks.Console/App.config

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<configuration>
+  <runtime>
+    <!-- https://docs.microsoft.com/en-us/dotnet/framework/configure-apps/file-schema/runtime/gcserver-element -->
+    <gcServer enabled="true" />
+    <!-- https://docs.microsoft.com/en-us/dotnet/framework/configure-apps/file-schema/runtime/gccpugroup-element -->
+    <GCCpuGroup enabled="true" />
+    <!-- https://docs.microsoft.com/en-us/dotnet/framework/configure-apps/file-schema/runtime/thread-useallcpugroups-element -->
+    <Thread_UseAllCpuGroups enabled="true"/>
+    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
+      <!-- Use codeBase entries for MSBuild assemblies so that the NGen'd native images will be used -->
+      <dependentAssembly>
+        <assemblyIdentity name="Microsoft.Build.Framework" culture="neutral" publicKeyToken="b03f5f7f11d50a3a" />
```

Comment:
Two reasons:

* Tasks loaded by MSBuild might have been compiled against older versions of MSBuild.  Since we're executing MSBuild 15+, these binding redirects make sure that user task assemblies will load properly even if they are compiled against older versions of MSBuild.
* We cannot ship MSBuild next to the application, so we need a way to locate it.  If we use assembly binding at runtime, the CLR will not choose the NGEN'd assemblies and JIT times are much slower.  Setting the basePaths to relative locations next to NuGet lets us load the NGen'd assemblies.

Reply:
Thanks, I'd be curious to understand how we can keep things in sync, but sounds you already pinged Rainer :)